app/enterprise/k8s-changelog.md

```diff
@@ -2,20 +2,36 @@
 title: Kong Enterprise k8s Changelog
 ---
 
+## 1.4.2.0
+
+> Released on 2020/01/02
+
+### Summary
+
+Kong Enterprise k8s 1.4.2.0 is based on Kong Gateway 1.4.2.
+
+### Kong Gateway
+
+Kong Enterprise k8s 1.4.2.0 is based on Kong Gateway 1.4.2,
+meaning it inherits all new features and changes between Kong Gateway 1.3.0 and
+1.4.2.
+Please checkout changelog for
+[Kong Gateway 1.4.0](https://github.com/Kong/kong/blob/master/CHANGELOG.md#140).
```

Comment:
is this correct?

Reply:
I'm not sure which part and what you are referring to here. Could you elaborate?

Reply:
the changlog pointing to 1.4.0, shouldn't it be [1.4.2](https://github.com/Kong/kong/blob/master/CHANGELOG.md#142)

Reply:
The reason I pointed it to 1.4.0 is that's the one that has most of the changes between 1.3.0 and 1.4.0. The two patch releases are just bug fixes and don't have anything notable there.